application/organisation-affiliation-data-manager/app.py

```diff
@@ -0,0 +1,83 @@
+from chalice import Chalice
+
+#   import json   //Packages to be imported once Chalice is deployed
+import boto3
+
+#   import uuid   //Packages to be imported once Chalice is deployed
+#   from boto3.dynamodb.conditions import Key //Packages to be imported once Chalice is deployed
+
+app = Chalice(app_name="organisation-affiliation-data-manager")
+dynamodb = boto3.resource("dynamodb")
+
+
+@app.route("/organisation_affiliation", methods=["GET"], cors=True)
```

Comment:
I think the recommended practice is for rest api resources/endpoints to be plurals eg organisations not organisation - so organisation_affiliations not organisation_affiliation. Happy to do this in my branch later if it helps
https://stackoverflow.com/questions/6845772/should-i-use-singular-or-plural-name-convention-for-rest-resources